qt/applications/workbench/workbench/widgets/about/test/test_about_presenter.py

```diff
@@ -169,3 +170,7 @@ def test_that_about_presenter_is_instantiated_without_error_when_getFacility_cau
 
         self.assertEqual(3, MockConfigService.getFacility.call_count)
         self.assertEqual(presenter._get_current_facility(), None)
+
+
+if __name__ == "__main__":
```

Comment:
Could we check this file is included in CMake? 
The fact this was missing implies were not potentially running the test on CI, as CMake will error if this is missing from a test file

Reply:
Yep I just checked and it's there, the tests are running on Jenkins too. I added this so that I could test this single file locally, and thought I'd leave it for that reason.

Reply:
Hmm, weird CMake should have checked for this missing line but either way it's out of scope for this PR. Thanks for looking into it